electricity/src/db.rs

```diff
@@ -0,0 +1,84 @@
+use aws_sdk_dynamodb::{
+    types::{
+        AttributeDefinition, ProvisionedThroughput, KeySchemaElement,
+        KeyType, ScalarAttributeType
+    },
+    config::{Region, Config},
+    operation::{create_table::CreateTableOutput},
+    Client, Error,
+};
+
+pub async fn make_config() -> Config {
+    let config = aws_config::from_env().profile_name("localstack").region(Region::new("us-west-2")).load().await;
+
+    aws_sdk_dynamodb::config::Builder::from(&config)
+        .endpoint_url(
+            "http://localhost:8000",
+        )
+        .build()
+}
+
+pub struct Database {
+    client: Option<Client>
+}
+
+impl Database {
+    pub async fn get_client(&mut self) -> &Client {
```

Comment:
I would rather:
- split a constructor and an global's variable init logic;
- use dotenvy to read settings and fallback to default "localhost";
- use either std::cell::OnceCell, crate once_cell or lazy_static to initialize global variable;
- do not use the global variable at all, and inject DB as a dependency into the collecting function.